frontend/src/pages/PainDiary.js

```diff
@@ -0,0 +1,45 @@
+import styled from 'styled-components'
+import Header from '../components/Header'
+import Content from '../components/Content'
+import Footer from '../components/Footer'
+import { useAuth } from '../auth/AuthProvider'
+import { useEffect, useState } from 'react'
+import { getPain } from '../service/AxiosService'
+import PainCard from '../components/PainCard'
+
+export default function PainDiary() {
+  const { token } = useAuth()
+  const [painList, setPainList] = useState([])
+
+  useEffect(() => {
+    getPain(token)
+      .then(response => response.data)
+      .then(data => setPainList(data.painList))
+  }, [token])
+
+  const painItems = painList.map(painItem => (
+    <PainCard
+      painLevel={painItem.painLevel}
+      date={painItem.date}
+      key={painItem.id}
+    />
+  ))
```

Comment:
würde ich in eine extra komponente auslagern

Reply:
die PainCard ist schon eine extra Komponente die Props übergeben bekommt. Oder meinst du die gesamte Funktion? Dann ist mir der Grund fürs auslagern nicht klar

Reply:
du könntest eine PainCardList komponente machen in der du dann die PainCards mapst dann wird das Paindiary nochmal aufgeräumter